libs/langchain-google-cloud-sql-pg/src/vectorStore.ts

```diff
@@ -180,55 +180,68 @@ class PostgresVectorStore extends VectorStore {
     )
   }
 
-  async addVectors(vectors: number[][], documents: DocumentInterface[], options?: { metadatas?: Array<Record<string, any>> | null, ids?: string[] }): Promise<string[] | void> {
-    if (options && (!options.ids || options.ids.length === 0)) {
-      options.ids = Array.from(documents).map(() => uuidv4());
-    }
+  async addVectors(vectors: number[][], documents: Document[], options?: { ids?: string[] }): Promise<string[] | void> {
+    let ids: string[] = [];
+    let metadatas: Record<string, any>[] = []
 
-    if (options && !options.metadatas) {
-      options.metadatas = Array.from(documents).map(() => new Object());
+    if (options && options.ids && options.ids.length === documents.length) {
```

Comment:
Can you throw an error if ids, vectors, and document lengths don't match like 
throw new Error(
        "The number of ids must match the number of vectors provided."
      );